src/integrationTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerControllerEndPointTest.java

```diff
@@ -39,10 +37,7 @@
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 import static uk.gov.hmcts.reform.wacaseeventhandler.helpers.InitiateTaskHelper.asJsonString;
 
-@ActiveProfiles({"local"})
```

Comment:
I think it's safe to move these lines to the parent class because all other IT will need them.